README.md

```diff
@@ -118,7 +132,39 @@ export default () => (
     <Loads cacheKey="randomDog" loadOnMount fn={getRandomDog}>
       {({ hasResponseInCache, isLoading, isSuccess, load, response }) => (
         <div>
+          {isIdle && <button onClick={load}>Load random dog</button>}
+          {isLoading && <div>loading...</div>}
+          {(isSuccess || hasResponseInCache) && (
+            <div>
+              {response && <img src={response.data.message} alt="Dog" />}
+              <div>
+                <button onClick={load}>Load another dog</button>
+              </div>
+            </div>
+          )}
+        </div>
+      )}
+    </Loads>
+  </LoadsProvider>
+);
+```
+
+Alternatively, localStorage can be utilised instead of caching data in the provider itself. Using localStorage is not currently compatible with the `loadOnMount` prop.
```

Comment:
Not compatible? How so?

Reply:
It will still work with a combination of a state boolean and `hasResponseInCache` right? I might work on a refactor for that before we ship local storage. Would like to get it right.

Reply:
It will work if you check hasResponseInCache -- yes -- but you cannot load it automatically the first time with `loadOnMount`. Doing so would overwrite the local storage cache on every mount.

Reply:
The result is the first load will have to be triggered by the app or user, and subsequent loads can be displayed from storage by checking hasResponseInCache.